test/extended/openstack/utils.go

```diff
@@ -46,6 +53,32 @@ type KuryrNetwork struct {
 	} `json:"status"`
 }
 
+type VolumeOption struct {
+	Name      string
+	PvcName   string
+	MountPath string
+}
+
+type DeploymentOpts struct {
```

Comment:
These structures are only used within this package for now, right?

```suggestion
type deploymentOpts struct {
```

Reply:
Done